runtime/src/apis.rs

```diff
@@ -167,21 +165,15 @@ impl_runtime_apis! {
 			frame_system::pallet::Pallet::<Self>::block_number()
 		}
 
-		fn get_random_rsa(block_hash: &[u8; 32]) -> (U512, U512) {
-			pallet_qpow::Pallet::<Self>::get_random_rsa(block_hash)
-		}
 
-		fn hash_to_group_bigint(h: &U512, m: &U512, n: &U512, solution: &U512) -> U512{
-			pallet_qpow::Pallet::<Self>::hash_to_group_bigint(h,m,n,solution)
-		}
-		fn get_max_distance() -> U512 {
-			pallet_qpow::Pallet::<Self>::get_max_distance()
+		fn get_max_difficulty() -> U512 {
```

Comment:
fwiw, another reminder that this will make the current version of the node incompatible with the new runtime if we don't properly version this. It's fine if we are going to reset the chain though